arrow-libs/core/arrow-core/src/main/kotlin/arrow/core/extensions/eval/monad/EvalMonad.kt

```diff
@@ -146,10 +146,7 @@ fun <A, B> Kind<ForEval, A>.followedBy(arg1: Kind<ForEval, B>): Eval<B> =
 )
 @Deprecated(
   "@extension kinded projected functions are deprecated",
-  ReplaceWith(
-    "apTap(arg1)",
-    "arrow.core.apTap"
-  ),
+  ReplaceWith("fix().flatMap { arg1.fix() }", "arrow.core.fix()"),
```

Comment:
should the ones that use `flatMap` and in general extension methods be included in the imports?
```suggestion
  ReplaceWith("fix().flatMap { arg1.fix() }", "arrow.core.fix", "arrow.core.flatMap"),
```
Also seems in this case `fix` is looking like `fix()` in the import.